emulator/src/rawdraw_sf.h

```diff
@@ -5020,11 +5020,13 @@ void CNFGGLXSetup( )
 		exit( -1 );
 	}
 	CNFGGLXFBConfig = cfgs[0];
+	XFree(cfgs);
 	XVisualInfo * vis = glXGetVisualFromFBConfig( CNFGDisplay, CNFGGLXFBConfig );
 	CNFGVisual = vis->visual;
 	CNFGVisualID = vis->visualid;
 	CNFGDepth = vis->depth;
 	CNFGCtx = glXCreateContext( CNFGDisplay, vis, NULL, True );
+	XFree(vis);
```

Comment:
This isn't the right way to make rawdraw changes. You should make them upstream at https://github.com/cntools/rawdraw/, including regenerating the single file header, have @cnlohr merge it, and copy the whole header back here. It's a hassle, but that's open source 🙂

Reply:
Boo, I guess I'll do it the right way... Looks like one of these was already in there already actually.

Reply:
In progress: https://github.com/cntools/rawdraw/pull/102

Reply:
Ah, I see one was already done. Neat.

Reply:
Yeah, when that's through update rawdraw here with the hash and all and we should be good to go.